spec/models/harvest_spec.rb

```diff
@@ -47,110 +47,110 @@
+      @harvest = FactoryBot.build(:harvest, weight_quantity: 0)
       @harvest.weight_quantity.should == 0
     end
 
     it "doesn't allow non-numeric weight quantities" do
-      @harvest = FactoryGirl.build(:harvest, weight_quantity: "99a")
+      @harvest = FactoryBot.build(:harvest, weight_quantity: "99a")
       @harvest.should_not be_valid
     end
   end
 
   context 'weight units' do
     it 'all valid units should work' do
       ['kg', 'lb', 'oz', nil, ''].each do |s|
-        @harvest = FactoryGirl.build(:harvest, weight_unit: s)
+        @harvest = FactoryBot.build(:harvest, weight_unit: s)
         @harvest.should be_valid
       end
     end
 
     it 'should refuse invalid weight unit values' do
-      @harvest = FactoryGirl.build(:harvest, weight_unit: 'not valid')
+      @harvest = FactoryBot.build(:harvest, weight_unit: 'not valid')
       @harvest.should_not be_valid
       @harvest.errors[:weight_unit].should include("not valid is not a valid unit")
     end
 
     it 'sets weight_unit to blank if quantity is blank' do
-      @harvest = FactoryGirl.build(:harvest, weight_quantity: '', weight_unit: 'kg')
+      @harvest = FactoryBot.build(:harvest, weight_quantity: '', weight_unit: 'kg')
       @harvest.should be_valid
       @harvest.weight_unit.should eq nil
     end
   end
 
   context "standardized weights" do
     it 'converts from pounds' do
-      @harvest = FactoryGirl.create(:harvest, weight_quantity: 2, weight_unit: "lb")
+      @harvest = FactoryBot.create(:harvest, weight_quantity: 2, weight_unit: "lb")
       @harvest.should be_valid
       @harvest.reload.si_weight.should eq 0.907
     end
 
     it 'converts from ounces' do
-      @harvest = FactoryGirl.create(:harvest, weight_quantity: 16, weight_unit: "oz")
+      @harvest = FactoryBot.create(:harvest, weight_quantity: 16, weight_unit: "oz")
       @harvest.should be_valid
       @harvest.reload.si_weight.should eq 0.454
     end
 
     it 'leaves kg alone' do
-      @harvest = FactoryGirl.create(:harvest, weight_quantity: 2, weight_unit: "kg")
+      @harvest = FactoryBot.create(:harvest, weight_quantity: 2, weight_unit: "kg")
       @harvest.should be_valid
       @harvest.reload.si_weight.should eq 2.0
     end
   end
 
   context 'ordering' do
     it 'lists most recent harvests first' do
-      @h1 = FactoryGirl.create(:harvest, created_at: 1.day.ago)
-      @h2 = FactoryGirl.create(:harvest, created_at: 1.hour.ago)
+      @h1 = FactoryBot.create(:harvest, created_at: 1.day.ago)
+      @h2 = FactoryBot.create(:harvest, created_at: 1.hour.ago)
       Harvest.all.should eq [@h2, @h1]
     end
   end
 
   context "stringification" do
-    let(:crop) { FactoryGirl.create(:crop, name: "apricot") }
+    let(:crop) { FactoryBot.create(:crop, name: "apricot") }
 
     it "apricots" do
-      @h = FactoryGirl.create(:harvest, crop: crop,
+      @h = FactoryBot.create(:harvest, crop: crop,
                                         quantity: nil,
```

Comment:
This is the kind of thing that Rubocop's complaining about - `quantity`, `unit` and `weight_quantity` should be lined up with `crop`, so lines 154-156 should be dedented by one space. You should be able to fix these all automatically using [`rubocop -a`](https://github.com/bbatsov/rubocop/wiki/Automatic-Corrections).

[I hate this indentation style, for exactly this reason, but it's the [Rubocop default](https://rubocop.readthedocs.io/en/latest/cops_layout/#layoutalignhash) so we've gone with it to ease transition from other Rails projects.]